ci_framework/plugins/action/discover_latest_image.py

```diff
@@ -39,26 +39,15 @@
 
 from ansible.errors import AnsibleError
 from ansible.plugins.action import ActionBase
-from ansible.module_utils.basic import missing_required_lib
 
 import re
 
-try:
-    from bs4 import BeautifulSoup
```

Comment:
I'd this library to removed from the dependencies too
@cescgina I think that this removal can simplify the adoption jobs you were preparing